include/swift/ABI/Task.h

```diff
@@ -513,6 +531,19 @@ class AsyncTask : public Job {
     return reinterpret_cast<GroupChildFragment *>(offset);
   }
 
+  // ==== Task Executor Preference --------------------------------------------
+
+  /// Returns true if the task has a task executor preference set,
+  /// specifically at creation time of the task. This may be from
+  /// inheriting the preference from a parent task, or by explicitly
+  /// setting it during creation (`Task(on:...)`).
+  ///
+  /// This means that during task tear down the record should be deallocated
+  /// because it was not set with a
```

Comment:
Another unfinished comme

Reply:
Thanks, fixed in 098ff1880ae0a3e1c713084e482919c1c63c8f01 and looked at all other comments as well.